django/bossspatialdb/views.py

```diff
@@ -292,6 +295,19 @@ def get(self, request, collection, experiment, channel):
                 channel_obj.save()
                 to_renderer["status"] = "DOWNSAMPLED"
 
+                # DP NOTE: Clear the cache of any cubes for the channel
+                #          This is to prevent serving stale data after
+                #          (re)downsampling
+                try:
+                    cache = RedisKVIO(settings.KVIO_SETTINGS)
+                    pipe = cache.cache_client.pipeline()
+                    for key in pipe.scan_iter("CACHED-CUBOID&"+lookup_key+"&*"):
```

Comment:
In `kvio.py`, `generate_cached_cuboid_keys()`, it looks like you could also have this prefix in Redis: `CACHED-CUBOID&ISO&`. We should clear those, too.

Reply:
Actually, thinking about a little more, how about creating a method to clear these keys in RedisKVIO itself? Then all the code related to the working with cache keys is closer together (file-wise).

Reply:
Since this is more like a temporary hack, that will be removed once the longer term solution is implemented, I wanted to keep the changes contained.
In the longer term solution there will be other changes to spdb so this can be rolled into that.

Reply:
Ok fair enough.